CorsixTH/Lua/languages/french.lua

```diff
@@ -1189,4 +1367,201 @@ map_editor_window = {
+}
+
+level_progress.hospital_value_enough =
+"Maintenez la valeur de votre hôpital au-dessus de %d et gérez vos autres problèmes pour remporter le niveau."
+level_progress.cured_enough_patients =
+"Vous avez guéri suffisamment de patients, mais vous devez améliorer votre hôpital pour remporter le niveau."
+
+
+-- Off you go, warnings
+
+-- Looking at the code it's doesn't seem to be needed, but it's shown as missing by the extract string process
+disease_discovered_patient_choice.need_to_employ = "Employez un(e) %s pour pouvoir gérer cette situation."
+multiplayer.everyone_failed =
+"Tout le monde a échoué à satisfaire le dernier objectif. Donc, tout le monde peut continuer à jouer !"
+multiplayer.players_failed = "Le(s) joueur(s) suivant(s) n'ont pas réussi à atteindre le dernier objectif :"
+original_credits[299] = " "
+original_credits[300] = " "
+original_credits[301] = ":Opérations"
+original_credits[302] = ","
+original_credits[303] = "Steve Fitton"
+original_credits[304] = " "
+original_credits[305] = " "
+original_credits[306] = " "
+original_credits[307] = ":Administration de la société"
+original_credits[308] = ","
+original_credits[309] = "Audrey Adams"
+original_credits[310] = "Annette Dabb"
+original_credits[311] = "Emma Gibbs"
+original_credits[312] = "Lucia Gobbo"
+original_credits[313] = "Jo Goodwin"
+original_credits[314] = "Sian Jones"
+original_credits[315] = "Kathy McEntee"
+original_credits[316] = "Louise Ratcliffe"
+original_credits[317] = " "
+original_credits[318] = " "
+original_credits[319] = " "
+original_credits[320] = ":Gestion de la société"
+original_credits[321] = ","
+original_credits[322] = "Les Edgar"
+original_credits[323] = "Peter Molyneux"
+original_credits[324] = "David Byrne"
+original_credits[325] = " "
+original_credits[326] = " "
+original_credits[327] = ":Toute l'équipe de Bullfrog Productions"
+original_credits[328] = " "
+original_credits[329] = " "
+original_credits[330] = " "
+original_credits[331] = ":Remerciements spéciaux à"
+original_credits[332] = ","
+original_credits[333] = "Tous ceux de l'hôpital de Frimley Park"
+original_credits[334] = " "
+original_credits[335] = ":Particulièrement"
+original_credits[336] = ","
+original_credits[337] = "Beverley Cannell"
+original_credits[338] = "Doug Carlisle"
+original_credits[339] = " "
+original_credits[340] = " "
+original_credits[341] = " "
+original_credits[342] = ":Continuez à réfléchir"
+original_credits[343] = " "
+original_credits[344] = " "
+original_credits[345] = " "
+original_credits[346] = " "
+original_credits[347] = " "
+original_credits[348] = " "
+original_credits[349] = " "
+original_credits[350] = " "
+original_credits[351] = " "
+original_credits[352] = " "
+original_credits[353] = " "
+original_credits[354] = " "
+original_credits[355] = " "
+original_credits[356] = " "
+original_credits[357] = " "
+original_credits[358] = " "
+original_credits[359] = " "
+original_credits[360] = " "
+original_credits[361] = "."
```

Comment:
Original credits are a bit weird.  I think each language was unique in the number of lines created.